### PR TITLE
Adds drink fluff; fixes drink descriptions

### PR DESCRIPTION
**Pull request by elgeonmb**
if something horribly wrong happens here it's because I'm deeply feverish

### CODE REVIEW

**Comment by Neerti** [comment]
``code/modules/examine/descriptions/containers.dm:53:error: unterminated text (expecting ")``

File: code/modules/examine/descriptions/containers.dm [hunk 1]
```diff
@@ -33,3 +33,66 @@
...
+	description_fluff = "Uncle Git's is a product of Abel's Rest, supply shortages and ubiqtuous observation during the occupation giving Takeshi 'Uncle' Git a chance to perfect this refinement of the common bathtub whiskey."
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/specialwhiskey
+	description_fluff = "Nobody knows what Special Blend Whiskey really is, how it's made, or where it comes from. It's an expensive Gilthari product, for only the most luxurious, but NanoTrasen managed to secure a supply deal."
+	
+/obj/item/weapon/reagent_containers/food/drinks/bottle/vodka
+	description_fluff = "Centauri Provisions claims that their vodka wins in blind taste-tests against traditional mixes from Earth, but they're not fooling anyone."
+	
+/obj/item/weapon/reagent_containers/food/drinks/bottle/tequilla
+	description_fluff = "This tequilla comes from the 'Cancer Craze' of the 20's, where consumer products would openly advertize their blatently carcinogenic nature to a burgeoning middle class with newfound access to cloned organs.'
```
Comment (Neerti) [inline code/modules/examine/descriptions/containers.dm]: Replace the ``'`` with a ``"`` at the end.

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 2]
```diff
@@ -319,7 +319,7 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/wine
 	name = "Doublebeard Bearded Special Wine"
-	desc = "A faint aura of unease and asspainery surrounds the bottle."
+	desc = "Cheap cooking wind pretending to be drinkable."
```
Comment (Neerti) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: It appears you misspelled wine.

File: code/modules/examine/descriptions/containers.dm [hunk 3]
```diff
@@ -33,3 +33,66 @@
...
+	description_fluff = "NanoTrasen makes this wine, which is generally regarded as better used for cooking than actually drinking. Morpheus specifically names Doublebeard in their recipe for Twobread."
+	
+/obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe
+	description_fluff = "A product of the anarchists of Casini's Reach, Jailebreaker Verte is an open-source synthesized liquour that has actually managed to succeed over traditional absinthes. Gilthari is livid."
+	
+/obj/item/weapon/reagent_containers/food/drinks/bottle/melonliquor
+	description_fluff = "Melon liquor, formerly an obscure drink, took off when young Skrell started dabbling in alcohol. This brand is owned by Vey-Med, one of their few diversified holdings."
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bluecuracao
+	description_fluff = "Centauri Provisions is currently marketting Miss Blue as the next big thing in mixed drinks. Analysts suggest that they're mistaken."
```
Comment (Atermonera) [inline code/modules/examine/descriptions/containers.dm]: Marketing with one 't'

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 4]
```diff
@@ -219,7 +219,7 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/vodka
 	name = "Tunguska Triple Distilled"
-	desc = "Aah, vodka. Prime choice of drink AND fuel by Russians worldwide."
+	desc = "Aah, vodka. Prime choice of drink and fuel by Russians worldwide."
```
Comment (Atermonera) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm]: Should probably be 'for Russians' while you're in there

**Pull request merged**